README.md

```diff
@@ -1,29 +1,29 @@
 # Profile
-This is a Sync! plugin for displaying pp real-time you are playing osu!std,you can use them for streamming or other ways.
+This is a Sync! plugin for displaying the current beatmap's pp in real-time when you are playing osu!, you can use this for streaming or etc!
 
 # Notice
-Please read Readme about [Osu!RTDP](https://github.com/KedamaOvO/OsuRTDataProvider-Release) and [OsuForum](https://osu.ppy.sh/forum/t/685031) before you use.
+Please read the README about [Osu!RTDP](https://github.com/KedamaOvO/OsuRTDataProvider-Release) and [OsuForum](https://osu.ppy.sh/forum/t/685031) before you use this.
```

Comment:
use `osu!RTDP` and `osu! forum thread`(consider use new web design link: "https://osu.ppy.sh/community/forums/topics/685031")